pyomo/opt/base/opt_config.py

```diff
@@ -9,17 +9,3 @@
 #  ___________________________________________________________________________
 
 import pyomo.common
-
```

Comment:
Where are these done now? Are they registered in the modules that use them?

Reply:
Explicit registration is not needed / recommended. They get “registered” the first time they are requested. By not registering executables, we avoid the file system search for programs we likely will never need.